modules/shared/partials/sdk-xattr-overview.adoc

```diff
@@ -1,18 +1,21 @@
 = Extended Attributes
 
 [abstract]
-Couchbase Server permits the definition of _extended attributes_.
+Extended Attributes (XATTR) are metadata that can be provided on a per-application basis.
+
+// tag::intro_extended_attributes[]
+Couchbase Server permits the definition of _extended attributes_ (XATTR).
```

Comment:
Is "definition" the right term to use here?  Feels like more "add" or something else.  

Related, where in the documentation does it cover the size of XATTRs and "system" XATTRs that are not virtual?

Reply:
Addition sounds good. :+1: 
Sizing note also added.